app/views/admin_posts/_navigation.html.erb

```diff
@@ -0,0 +1,15 @@
+<h3 class="landmark heading"><%= t(".landmark") %></h3>
+<% if policy(AdminPost).can_post? %>
```

Comment:
Only admins who `can_post?` are able to access the pages where this navigation appears, so this check probably isn't necessary -- especially since we know the logic for this navigation will need reworking when we add news post drafts in [AO3-6960](https://otwarchive.atlassian.net/issues/AO3-6960).